metron-analytics/metron-profiler-spark/README.md

```diff
@@ -42,25 +42,8 @@ The portion of a profile produced by the Batch Profiler should be indistinguisha
 For an introduction to the Profiler, see the [Profiler README](../metron-profiler-common/README.md).
 
 ## Getting Started
-
-1. Create a profile definition by editing `$METRON_HOME/config/zookeeper/profiler.json` as follows.  
-
-    ```
-    cat $METRON_HOME/config/zookeeper/profiler.json
-    {
-      "profiles": [
-        {
-          "profile": "hello-world",
-          "foreach": "'global'",
-          "init":    { "count": "0" },
-          "update":  { "count": "count + 1" },
-          "result":  "count"
-        }
-      ],
-      "timestampField": "timestamp"
-    }
-    ```
-
+1. Chose if you are going to use a profile already loaded into zookeeper or if you are going to use a profile defined in a JSON file on disk. See [Specifying profiles](#specifying-profiles) for more information.
+ 
```

Comment:
Can you restore the original step 1 as it was?  For a "Getting Started" section, I think it makes sense to give a user specific steps to see something happen.  With this change, we don't have that.

I think you could just append your last sentence to that.  Maybe something like this.

> 1. Create a profile definition by editing `$METRON_HOME/config/zookeeper/profiler.json` as follows. See [Specifying Profiles](#specifying-profiles) for more information on how profiles can be defined.

Reply:
Done.